packages/renderer/src/ffmpeg-flags.ts

```diff
@@ -57,3 +62,89 @@ export const getFfmpegVersion = async (options: {
 	});
 	return parseFfmpegVersion(buildInfo);
 };
+
+export const downloadFfmpeg = async (): Promise<void> => {
+	// implement callback instead
+	function onProgress(downloadedBytes: number, totalBytes: number) {
+		console.log(
+			'Downloading ffmpeg: ',
+			toMegabytes(downloadedBytes) + '/' + toMegabytes(totalBytes)
+		);
+	}
+
+	const os = require('os');
+	const path = require('path');
+	const destinationPath =
+		os.platform() === 'win32'
+			? path.resolve(process.cwd(), 'node_modules/.ffmpeg/ffmpeg.exe')
+			: path.resolve(process.cwd(), 'node_modules/.ffmpeg/ffmpeg');
```

Comment:
- Lot of redundancy here, would be better to make a function that returns the folder name and one that returns the absolute path, and then use it across the PR, for writing and reading the file
- More precise than process.cwd() would be `remotionRoot` which you should be able to access by passing it as an argument through the functions. `remotionRoot` is like process.cwd() but it handles the case that the user started remotion from a directory not adjacent to a package.json